### PR TITLE
[improve] : Delete Chinese comments, modify word errors, add Operatio…

### PR DESCRIPTION
**Pull request by Yanshuming1**
[improve] : Delete Chinese comments, modify word errors, add Operatio…

### CODE REVIEW

File: manager/src/main/resources/application.yml [hunk 1]
```diff
@@ -186,7 +186,7 @@ alerter:
   telegram-webhook-url: https://api.telegram.org/bot%s/sendMessage
   # discord
   discord-webhook-url: https://discord.com/api/v9/channels/%s/messages
-  # server酱
+  # server
```
Comment (tomsun28) [inline manager/src/main/resources/application.yml:189]: hi, this english is serverChan

File: manager/src/main/resources/sureness.yml [hunk 2]
```diff
@@ -113,7 +113,7 @@ account:
     role: [guest]
   - appId: lili
     # credential = MD5(password + salt)
-    # plain password: hertzbeat
+    # plain password: hertzBeat
```
Comment (tomsun28) [inline manager/src/main/resources/sureness.yml:116]: hi wrong password

File: manager/src/main/java/org/apache/hertzbeat/manager/controller/AiController.java [hunk 3]
```diff
@@ -44,6 +46,9 @@ public class AiController {
     @Autowired
     private AiServiceFactoryImpl aiServiceFactory;
 
+    /**
+     * Types of artificial intelligence
+     */
     @Value("${aiConfig.type:0}")
```
Comment (tomsun28) [inline manager/src/main/java/org/apache/hertzbeat/manager/controller/AiController.java:52]: type default is not a 0

File: manager/src/main/resources/application.yml (hunk 1 above)
Comment (Yanshuming1) [reply manager/src/main/resources/application.yml]: fix

File: manager/src/main/resources/sureness.yml (hunk 2 above)
Comment (Yanshuming1) [reply manager/src/main/resources/sureness.yml]: fix

File: manager/src/main/java/org/apache/hertzbeat/manager/controller/AiController.java (hunk 3 above)
Comment (Yanshuming1) [reply manager/src/main/java/org/apache/hertzbeat/manager/controller/AiController.java:52]: https://github.com/apache/hertzbeat/pull/2229：In this branch has been modified, please merge this branch first, and then I will merge type

Comment (Yanshuming1) [reply manager/src/main/java/org/apache/hertzbeat/manager/controller/AiController.java]: fix

**Comment by tomsun28** [comment]
hi, when i am not config the ai prop, startup error happen when start the server. Suggest not error when user not config the ai prop. 

```
Error starting ApplicationContext. To display the condition evaluation report re-run your application with 'debug' enabled.
2024-07-07 22:28:07 [main] ERROR org.springframework.boot.SpringApplication - Application run failed
org.springframework.beans.factory.UnsatisfiedDependencyException: Error creating bean with name 'aiController': Unsatisfied dependency expressed through field 'aiServiceFactory': Error creating bean with name 'aiServiceFactoryImpl': Unsatisfied dependency expressed through field 'aiService': Error creating bean with name 'SparkDeskAiServiceImpl': Injection of autowired dependencies failed
	at org.springframework.beans.factory.annotation.AutowiredAnnotationBeanPostProcessor$AutowiredFieldElement.resolveFieldValue(AutowiredAnnotationBeanPostProcessor.java:787)
	at org.springframework.beans.factory.annotation.AutowiredAnnotationBeanPostProcessor$AutowiredFieldElement.inject(AutowiredAnnotationBeanPostProcessor.java:767)
	at org.springframework.beans.factory.annotation.InjectionMetadata.inject(InjectionMetadata.java:145)
	at org.springframework.beans.factory.annotation.AutowiredAnnotationBeanPostProcessor.postProcessProperties(AutowiredAnnotationBeanPostProcessor.java:508)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.populateBean(AbstractAutowireCapableBeanFactory.java:1419)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.doCreateBean(AbstractAutowireCapableBeanFactory.java:599)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.createBean(AbstractAutowireCapableBeanFactory.java:522)
	at org.springframework.beans.factory.support.AbstractBeanFactory.lambda$doGetBean$0(AbstractBeanFactory.java:325)
	at org.springframework.beans.factory.support.DefaultSingletonBeanRegistry.getSingleton(DefaultSingletonBeanRegistry.java:234)
	at org.springframework.beans.factory.support.AbstractBeanFactory.doGetBean(AbstractBeanFactory.java:323)
	at org.springframework.beans.factory.support.AbstractBeanFactory.getBean(AbstractBeanFactory.java:199)
	at org.springframework.beans.factory.support.DefaultListableBeanFactory.preInstantiateSingletons(DefaultListableBeanFactory.java:975)
	at org.springframework.context.support.AbstractApplicationContext.finishBeanFactoryInitialization(AbstractApplicationContext.java:959)
	at org.springframework.context.support.AbstractApplicationContext.refresh(AbstractApplicationContext.java:624)
	at org.springframework.boot.web.servlet.context.ServletWebServerApplicationContext.refresh(ServletWebServerApplicationContext.java:146)
	at org.springframework.boot.SpringApplication.refresh(SpringApplication.java:754)
	at org.springframework.boot.SpringApplication.refreshContext(SpringApplication.java:456)
	at org.springframework.boot.SpringApplication.run(SpringApplication.java:334)
	at org.springframework.boot.SpringApplication.run(SpringApplication.java:1354)
	at org.springframework.boot.SpringApplication.run(SpringApplication.java:1343)
	at org.apache.hertzbeat.manager.Manager.main(Manager.java:44)
Caused by: org.springframework.beans.factory.UnsatisfiedDependencyException: Error creating bean with name 'aiServiceFactoryImpl': Unsatisfied dependency expressed through field 'aiService': Error creating bean with name 'SparkDeskAiServiceImpl': Injection of autowired dependencies failed
	at org.springframework.beans.factory.annotation.AutowiredAnnotationBeanPostProcessor$AutowiredFieldElement.resolveFieldValue(AutowiredAnnotationBeanPostProcessor.java:787)
	at org.springframework.beans.factory.annotation.AutowiredAnnotationBeanPostProcessor$AutowiredFieldElement.inject(AutowiredAnnotationBeanPostProcessor.java:767)
	at org.springframework.beans.factory.annotation.InjectionMetadata.inject(InjectionMetadata.java:145)
	at org.springframework.beans.factory.annotation.AutowiredAnnotationBeanPostProcessor.postProcessProperties(AutowiredAnnotationBeanPostProcessor.java:508)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.populateBean(AbstractAutowireCapableBeanFactory.java:1419)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.doCreateBean(AbstractAutowireCapableBeanFactory.java:599)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.createBean(AbstractAutowireCapableBeanFactory.java:522)
	at org.springframework.beans.factory.support.AbstractBeanFactory.lambda$doGetBean$0(AbstractBeanFactory.java:325)
	at org.springframework.beans.factory.support.DefaultSingletonBeanRegistry.getSingleton(DefaultSingletonBeanRegistry.java:234)
	at org.springframework.beans.factory.support.AbstractBeanFactory.doGetBean(AbstractBeanFactory.java:323)
	at org.springframework.beans.factory.support.AbstractBeanFactory.getBean(AbstractBeanFactory.java:199)
	at org.springframework.beans.factory.config.DependencyDescriptor.resolveCandidate(DependencyDescriptor.java:254)
	at org.springframework.beans.factory.support.DefaultListableBeanFactory.doResolveDependency(DefaultListableBeanFactory.java:1443)
	at org.springframework.beans.factory.support.DefaultListableBeanFactory.resolveDependency(DefaultListableBeanFactory.java:1353)
	at org.springframework.beans.factory.annotation.AutowiredAnnotationBeanPostProcessor$AutowiredFieldElement.resolveFieldValue(AutowiredAnnotationBeanPostProcessor.java:784)
	... 20 common frames omitted
Caused by: org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'SparkDeskAiServiceImpl': Injection of autowired dependencies failed
	at org.springframework.beans.factory.annotation.AutowiredAnnotationBeanPostProcessor.postProcessProperties(AutowiredAnnotationBeanPostProcessor.java:514)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.populateBean(AbstractAutowireCapableBeanFactory.java:1419)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.doCreateBean(AbstractAutowireCapableBeanFactory.java:599)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.createBean(AbstractAutowireCapableBeanFactory.java:522)
	at org.springframework.beans.factory.support.AbstractBeanFactory.lambda$doGetBean$0(AbstractBeanFactory.java:325)
	at org.springframework.beans.factory.support.DefaultSingletonBeanRegistry.getSingleton(DefaultSingletonBeanRegistry.java:234)
	at org.springframework.beans.factory.support.AbstractBeanFactory.doGetBean(AbstractBeanFactory.java:323)
	at org.springframework.beans.factory.support.AbstractBeanFactory.getBean(AbstractBeanFactory.java:199)
	at org.springframework.beans.factory.config.DependencyDescriptor.resolveCandidate(DependencyDescriptor.java:254)
	at org.springframework.beans.factory.support.DefaultListableBeanFactory.addCandidateEntry(DefaultListableBeanFactory.java:1689)
	at org.springframework.beans.factory.support.DefaultListableBeanFactory.findAutowireCandidates(DefaultListableBeanFactory.java:1653)
	at org.springframework.beans.factory.support.DefaultListableBeanFactory.resolveMultipleBeanCollection(DefaultListableBeanFactory.java:1543)
	at org.springframework.beans.factory.support.DefaultListableBeanFactory.resolveMultipleBeans(DefaultListableBeanFactory.java:1511)
	at org.springframework.beans.factory.support.DefaultListableBeanFactory.doResolveDependency(DefaultListableBeanFactory.java:1392)
	at org.springframework.beans.factory.support.DefaultListableBeanFactory.resolveDependency(DefaultListableBeanFactory.java:1353)
	at org.springframework.beans.factory.annotation.AutowiredAnnotationBeanPostProcessor$AutowiredFieldElement.resolveFieldValue(AutowiredAnnotationBeanPostProcessor.java:784)
	... 34 common frames omitted
Caused by: java.lang.IllegalArgumentException: Could not resolve placeholder 'aiConfig.api-secret' in value "${aiConfig.api-secret}"
	at org.springframework.util.PropertyPlaceholderHelper.parseStringValue(PropertyPlaceholderHelper.java:180)
	at org.springframework.util.PropertyPlaceholderHelper.replacePlaceholders(PropertyPlaceholderHelper.java:126)
	at org.springframework.core.env.AbstractPropertyResolver.doResolvePlaceholders(AbstractPropertyResolver.java:239)
	at org.springframework.core.env.AbstractPropertyResolver.resolveRequiredPlaceholders(AbstractPropertyResolver.java:210)
	at org.springframework.context.support.PropertySourcesPlaceholderConfigurer.lambda$processProperties$0(PropertySourcesPlaceholderConfigurer.java:200)
	at org.springframework.beans.factory.support.AbstractBeanFactory.resolveEmbeddedValue(AbstractBeanFactory.java:921)
	at org.springframework.beans.factory.support.DefaultListableBeanFactory.doResolveDependency(DefaultListableBeanFactory.java:1374)
	at org.springframework.beans.factory.support.DefaultListableBeanFactory.resolveDependency(DefaultListableBeanFactory.java:1353)
	at org.springframework.beans.factory.annotation.AutowiredAnnotationBeanPostProcessor$AutowiredFieldElement.resolveFieldValue(AutowiredAnnotationBeanPostProcessor.java:784)
	at org.springframework.beans.factory.annotation.AutowiredAnnotationBeanPostProcessor$AutowiredFieldElement.inject(AutowiredAnnotationBeanPostProcessor.java:767)
	at org.springframework.beans.factory.annotation.InjectionMetadata.inject(InjectionMetadata.java:145)
	at org.springframework.beans.factory.annotation.AutowiredAnnotationBeanPostProcessor.postProcessProperties(AutowiredAnnotationBeanPostProcessor.java:508)
	... 49 common frames omitted
Disconnected from the target VM, address: '127.0.0.1:58206', transport: 'socket'
```

**Comment by tomsun28** [comment]
```
    @Value("${aiConfig.model:generalv3.5}")
    private String model;

    @Value("${aiConfig.api-key}")
    private String apiKey;
    @Value("${aiConfig.api-secret}")
    private String apiSecret;
```
hi suggest not use Properties like this, the case where no value must be considered.

**Comment by tomsun28** [comment]
```
@Service("SparkDeskAiServiceImpl")
@Slf4j
public class SparkDeskAiServiceImpl implements AiService {
```
you can try to use `@ConditionalOnProperty` to decide whether to initialize the bean, refer the `warehouse`

```
@Component
@ConditionalOnProperty(prefix = "warehouse.store.iot-db", name = "enabled", havingValue = "true")
@Slf4j
public class IotDbDataStorage extends AbstractHistoryDataStorage {
```

**Comment by tomsun28** [comment]
```
# AI config
aiConfig:
  # AI Type:zhiPu、alibabaAi、kimiAi、sparkDesk
  type:
  # Model name：glm-4、qwen-turboo、moonshot-v1-8k、generalv3.5
  model:
  # api key
  api-key:
  #At present, only IFLYTEK large model needs to be filled in
  api-secret:
```
suggest use the `ai` instead of `aiConfig`, keep the naming consistent with other configuration properties. `xxxConfig` is not need.

**Review by zqr10159 — APPROVED** [review]

**Pull request merged**